index.html

```diff
@@ -9,6 +9,29 @@
         <link rel="stylesheet" href="main.css">
     </head>
     <body>
-        
+        <nav>
+            <p>Welcome</p>
+            <a href="#"><img src="img/Icon-Menu.svg" alt="Menu"></a>
+        </nav>
+        <section class="headline">
+            <header>
+                <h2>Hey there. I'm Saadat 
+                    <br>
+                    <span>I'm a software developer</span>
+                </h2> 
```

Comment:
- [ ] Consider using `h1` tag here instead of `h2` tag. This is because this is the first content a user should see when they enter your page and also to improve the semantic structure of your HTML. You can also remove the `span` tag as the `br` tag is enough to push it to the next line. See suggestion:

```suggestion
                <h1>Hey there. I'm Saadat 
                    <br>
                    I'm a software developer
                </h1> 
```

Reply:
Thanks for identifing the issue @julie-ify. But I have a question. If I remove span tag how can I apply different color to "I'm a software developer"?